### ISSUE TITLE
Notify user when trying to create a lesson with no notes

### ISSUE THREAD

**Issue by jrobind** [jrobind/dev-jot#64]
Currently, a user is prevented from creating a lesson that contains no text. Even though the validation logic is correct there isn't a message or visual cue to inform the user the lesson creation has failed. 

![ezgif com-gif-maker (5)](https://user-images.githubusercontent.com/30320536/102106567-a69f3e00-3e28-11eb-98ae-131f0176dc98.gif)

**Comment by edXmO** [comment]
I think, there's different approaches for this:
* Notify the user via pop-up 
* Modify the text within the fields: "Can't add empty lessons."
* Add some different style to the empty field, so the user notices. Maybe change the border. 

Maybe, the better option is the less invasive for the user. 

What do you think?

**Comment by jrobind** [comment]
I think at a minimum we should at least change the border colour. The native HTML input validation renders a tooltip when trying to submit without a lesson title like so:

![Screen Shot 2020-12-17 at 12 01 46](https://user-images.githubusercontent.com/30320536/102485570-b6549780-405f-11eb-8f4f-22d626efbd8a.png)

I think changing the border colour is something we should definitely do – maybe adding a tooltip/pop-up is a nice-to-have. You could take that on if you wanted to? Or maybe add the border colour for validation for this issue and tackle a tooltip/pop-up in another, follow-up issue ?

**Comment by edXmO** [comment]
Yes, I'll give it a go. I'll first focus on adding the border colour in the respective use cases and then maybe address the tooltip/pop-up.

**Comment by jrobind** [comment]
Awesome! Sounds good 👍

**Comment by SakaRicky** [comment]
Hi, can I look into this issue?

**Comment by jrobind** [comment]
> Hi, can I look into this issue?

Hey! Unfortunately this issue is already being worked on.

**Comment by SakaRicky** [comment]
Looking for a first issue to contribute here. Can you point me to a good one please?

**Comment by jrobind** [comment]
> Looking for a first issue to contribute here. Can you point me to a good one please?

Take a look at #113. That's a good one to start with. If you want I can assign you to it?

**Comment by SakaRicky** [comment]
Looks good for me. I finally have the opportunity to tackle SASS/SCSS

**Comment by jrobind** [comment]
> Looks good for me. I finally have the opportunity to tackle SASS/SCSS

😄 just ping on the ticket if you need a hand. Do as little or as much as you want. Whatever you're comfortable with!

**Comment by SakaRicky** [comment]
I see there are 2 versions. I want to work with the react version. Can I clone that version only? Which version does git clone https://github.com/jrobind/dev-jot.git clone ?

**Comment by jrobind** [comment]
This ticket is just for the vanilla app. The react version uses styled components and material ui so this ticket doesn't apply.

**Comment by Aslam-13** [comment]
Hi,
I have made some changes in the ui can i make a pull request!
